Trappist/src/Promact.Trappist.Repository/Test/TestsRepository.cs

```diff
@@ -15,10 +22,18 @@ public class TestsRepository : ITestsRepository
     {
         private readonly TrappistDbContext _dbContext;
         private readonly IGlobalUtil _util;
-        public TestsRepository(TrappistDbContext dbContext, IGlobalUtil util)
+        private readonly IStringConstants _stringConstants;
+        private QuestionAC questionAc;
+        private TestAC testACObject;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the 'testACObject' field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)